bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java

```diff
@@ -205,6 +206,12 @@ private List<String> addJsonParseCommand()
 		return arguments;
 	}
 
+	public double parseVersionWithMultipleDotsToDouble(String version)
+	{
+		String numericVersion = version.replace(".", ""); //$NON-NLS-1$ //$NON-NLS-2$
+		return Double.parseDouble(numericVersion) / Math.pow(10, version.split("\\.").length - 1.0); //$NON-NLS-1$
+	}
+
```

Comment:
I would suggest to replace with the Apache ComparableVersion utility for doing this version comparisions.

https://mvnrepository.com/artifact/org.apache.maven/maven/3.9.5


```
public boolean isVersionAtLeast(String currentIDFVersion, String minimumIDFVersion) {
    ComparableVersion currentVersion = new ComparableVersion(currentIDFVersion);
    ComparableVersion minVersion = new ComparableVersion(minimumIDFVersion);
    return currentVersion.compareTo(minVersion) >= 0;
}
```

Reply:
done